recipes/prophet/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "prophet" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  binary_has_prefix_files:
+    - Lib/site-packages/prophet/stan_model/prophet_model.pkl          # [win]
+    - lib/python*/site-packages/prophet/stan_model/prophet_model.pkl  # [unix]
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - pandas >=1.0.4
+    - pystan ==2.19.1.1
+    - LunarCalendar
+    - convertdate
+    - holidays >=0.10.2
+    - python-dateutil
+    - tqdm
+    - numpy
```

Would drop anything that doesn't need to be used for the build here

```suggestion
    - python
    - pip
```

---

(answered below)